src/js/jsx/search/SearchBar.jsx

```diff
@@ -429,12 +443,16 @@ define(function (require, exports, module) {
                 category = "category-" + type;
 
             // Seeing if filters are active or not for analytics
-            if (filters.length === 0) {
+            if (filters && filters.size === 0) {
                 filtersString = "filter-inactive";
             } else {
                 filtersString = "filter-active";
             }
-            
+
+            var flux = this.getFlux(),
+                searchStore = flux.store("search"),
+                payload = searchStore.getPayloadFromFilter(lastFilter);
```

Comment:
I guess this doesn't need to be changed here, but `payload` is not a very good word to use here because it's not descriptive at all. If I'm not familiar with this code, I'm scratching my head thinking "what the heck is a payload?". Ideally there would be a more descriptive term for whatever this function returns that makes it possible for me to keep reading without having to pause and go read that function to see roughly what the heck it does..